Libplanet/Blockchain/Renderers/IActionRenderer.cs

```diff
@@ -0,0 +1,104 @@
+#nullable enable
+using System;
+using Libplanet.Action;
+
+namespace Libplanet.Blockchain.Renderers
+{
+    /// <summary>
+    /// Listens state changes of every step of actions, besides blocks, on a
+    /// <see cref="BlockChain{T}"/>.  If you need more fine-grained events than
+    /// <see cref="IRenderer{T}"/> implement this interface instead.
```

Comment:
Doesn't it need `,` after *if* usage?

```suggestion
    /// <see cref="IRenderer{T}"/>, implement this interface instead.
```

Reply:
Amended!